app/celery/tasks.py

```diff
@@ -20,7 +20,7 @@
     notify_celery,
 )
 from app.aws import s3
-from app.celery import provider_tasks, letters_pdf_tasks, research_mode_tasks
+from app.celery import provider_tasks, research_mode_tasks
 from app.config import QueueNames
 from app.dao.daily_sorted_letter_dao import dao_create_or_update_daily_sorted_letter
```

Comment:
Shouldn't this import be removed? `dao_create_or_update_daily_sorted_letter`

Reply:
It's used once later in this file starting at line 460 in `def persist_daily_sorted_letter_counts()` - shouldn't that also be removed?

Reply:
There are in fact other letters methods in this file - why are we keeping those?